src/functions/template-tags/general.php

```diff
@@ -1355,6 +1355,13 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 		 */
 		$allow_shortcode = apply_filters( 'tribe_events_excerpt_allow_shortcode', false );
 
+		/**
+		 * Prevent the removal of shortcode markup in the Excerpt
```

Comment:
I wonder if we should make it clear that this will remove anything that even _looks_ like a shortcode, just to help people understand possible reasons to use it (such as if they have things like _"blah blah [footnote 5]"_ in their copy)?